test/quoteSpec.js

```diff
@@ -362,6 +364,44 @@ describe('Quotes', function () {
         .end()
     })
 
+    it('caches source and destination ledger precision', function * () {
+      nock.cleanAll()
+      nock('http://eur-ledger.example/EUR')
+        .get('').reply(200, {precision: 10, scale: 4})
+        .get('').reply(500, 'Invalid request')
```

Comment:
I can't find docs for `done` behavior

On Wed, Mar 30, 2016 at 12:10 PM, Chris Clark <notifications@github.com>
wrote:

> In test/quoteSpec.js
> <https://github.com/interledger/five-bells-connector/pull/132#discussion_r57945792>
> :
>
> > @@ -362,6 +364,44 @@ describe('Quotes', function () {
> >          .end()
> >      })
> >
> > +    it('caches source and destination ledger precision', function * () {
> > +      nock.cleanAll()
> > +      nock('http://eur-ledger.example/EUR')
> > +        .get('').reply(200, {precision: 10, scale: 4})
> > +        .get('').reply(500, 'Invalid request')
>
> node-nock/nock#415 <https://github.com/node-nock/nock/issues/415>
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/interledger/five-bells-connector/pull/132/files/10ef121741572e5d919bf31c508da65451d77203#r57945792>
>